doh.c

```diff
@@ -714,6 +717,7 @@ int main(int argc, char **argv)
   int exit_status = 0;
   int queued;
   int url_argc = 1;
+  long insecure_mode = 1L;
```

Comment:
`test_mode` was recently added and is a `int`. I think it would make more sense to use that type, if we don't go full `bool`...

Reply:
I used a long because I could then pass it directly to `curl_easy_setopt`, but I can change this to a `bool` in anticipation of changing the rest of them.